### DISCUSSION TITLE
the demos run too slow

### DISCUSSION THREAD

**Discussion by ShaYito** [InteractiveComputerGraphics/PositionBasedDynamics#111] [Q&A]
I am using windows 10, vs 2019 with RTX 3060, i7-11800H, and 24G memory.
I ran the demo "DeformableCollisionDemo" and it takes about 30 seconds for the object to first fall on the torus. It seems too slow.
And I notice the demo only uses GPU when the scene appears and after that GPU is not utilized. (and the CPU was 100% taken up)
Other demos are running too slow as well.
Is this normal?

**Comment by janbender** [comment]
No. This even runs fast on my slow laptop. Did you compile in Release mode?

**Reply by ShaYito** [reply]
Thank you so much.😀 I switched to Release mode and it runs so fast now. I didn't know Debug mode would slow down the program.

**Reply by janbender** [reply]
In Debug mode the code is compiled without any optimization.

**Reply by dandingol03** [reply]
how to change to release mode

**Reply by janbender** [reply]
Depends on your system. For Linux CMake should automatically set the Release mode as default. For using a IDE like Visual Studio you can set this in the configuration manager.
